content/docs/gke/deploy/deploy-cli.md

```diff
@@ -61,6 +61,7 @@ Follow these steps to deploy Kubeflow:
     ```bash
     # The following command is optional, to make kfctl binary easier to use.
     export PATH=$PATH:<path to kfctl in your kubeflow installation>
+    export ZONE=<your target zone> #where the deployment will be created
```

Comment:
We should give a bit more of a clue that "zone" is a GCP concept:

```
export ZONE=<your target GCP zone>
```

Reply:
I agree Sarah. 
Since the page in question is for GKE:
https://www.kubeflow.org/docs/gke/deploy/deploy-cli/

Probably it gives some context. But erring on being more explicit is always better.
Do I need to create another pull request or can it be accepted with your change?

Reply:
Thanks @ultrons! You can use this PR. If you make the change in the same branch, GitHub automatically adds the change to this PR. Since it's a small change, you can do it in the GitHub UI if you prefer. If that doesn't work, let me know and I'll approve this PR then you can make another.

Reply:
Friendly ping @ultrons